pkg/harness/stackrun/v1/types.go

```diff
@@ -72,12 +77,21 @@ func (in *StackRun) Env() []string {
 		env[i] = fmt.Sprintf("%s=%s", e.Name, e.Value)
 	}
 
+	var tfArgs []string
 	if in.Creds != nil {
```

Comment:
why are you wiping PLURAL_ACCESS_TOKEN?  That'll also crash the process if it needs plural creds